plugins/traction_innkeeper/traction_innkeeper/v1_0/tenant/__init__.py

```diff
@@ -4,11 +4,50 @@
 from aries_cloudagent.core.event_bus import EventBus
 from aries_cloudagent.core.plugin_registry import PluginRegistry
 from aries_cloudagent.core.protocol_registry import ProtocolRegistry
+from aries_cloudagent.config.logging import TimedRotatingFileMultiProcessHandler
+
+from pythonjsonlogger import jsonlogger
 
 from .holder_revocation_service import subscribe, HolderRevocationService
+from .routes import context_tenant_id
 
 LOGGER = logging.getLogger(__name__)
 
+LOG_FORMAT_FILE_ALIAS_PATTERN = (
+    "%(asctime)s %(tenant_id)s %(levelname)s %(pathname)s:%(lineno)d %(message)s"
+)
+
+
+class ContextFilter(logging.Filter):
+    """Custom logging filter to adapt logs with contextual tenant_id."""
+
+    def __init__(self):
+        """Initialize an instance of logging filter."""
+        super(ContextFilter, self).__init__()
+
+    def filter(self, record):
+        """Filter log records and add tenant id to them."""
+        try:
+            tenant_id = context_tenant_id.get()
+            record.tenant_id = tenant_id
+        except LookupError:
+            record.tenant_id = None
+
+        return True
```

Comment:
This needs to be here?

Reply:
Yeah it basically allows every log to be included